pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2827,7 +2827,33 @@
       ?~  url=(slaw %t i.t.t.tyl)        [~ ~]
       ?~  entry=(~(get by cache) u.url)  [~ ~]
       ?.  =(u.aeon aeon.u.entry)         [~ ~]
-      ``noun+!>(val.u.entry)
+      ?~  val.u.entry                    [~ ~]
+      =/  val  u.val.u.entry
+      ?:  ?=(%payload -.body.val)
+        ``noun+!>(val)
+      ::  body is a scry; scry it to get a payload
+      ::
+      ?~  path.body.val                  [~ ~]
+      ?~  bem=(de-beam t.path.body.val)  [~ ~]
+      =/  res=(unit (unit cage))  (rof ~ i.path.body.val u.bem)
+      ?~  res    [~ ~]
+      ?~  u.res  [~ ~]
+      =*  vase   q.u.u.res
+      ?.  ?=(@ q.vase)  [~ ~]
+      =/  data  (as-octs:mimes:html q.vase)
```

Comment:
Eyre should mark-convert this to `%mime` instead of asserting it's an atom and then calling `+as-octs` manually.  Clay files can all be converted to `%mime`.  It will need to scry a `$tube` out of Clay based on the mark found at `p.u.u.res`, scrying it out of the desk found at `q.u.bem` with a `%cc` or `%cf` scry, whichever is easier to use.  A `%cc` scry returns a vase of a `$tube` gate that takes in the vase of an input and returns a vase of a mark-converted output; `%cf` returns a vase of a statically typed gate that takes in an input of one mark and outputs a statically typed value of another mark.

I think `%cf` is better.  Code would look something like this:
```hoon
=+  .^(cast=vase %cf (en-beam -.u.bem /[p.u.u.res]/mime))
(slam cast q.u.u.res)
```

Reply:
Having trouble getting this to work, because `%cf` returns a static gate, not a vase. `+sign-to-json` uses `%cf` with `rof` (rather than a raw `.^`) and that seems to work, but when I try it in `scry-to-mime` it fails with `clay: read-at-aeon fail [desk=%$ care=%f case=[%da p=~2023.3.18..16.53.39..d4f9] path=/noun/mime]`. 😕